### PR TITLE
Clarify style rule structure of ProfileSettings to avoid unexpected regressions

### PR DESCRIPTION
**Pull request by luixxiul**
This PR gives an order to the style rules structure of `ProfileSettings` to avoid unexpected regressions.

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

type: task

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: src/components/views/room_settings/RoomProfileSettings.tsx [hunk 1]
```diff
@@ -270,8 +271,10 @@ export default class RoomProfileSettings extends React.Component<IProps, IState>
                             disabled={!this.state.canSetName}
                         />
                         <Field
-                            className="mx_ProfileSettings_controls_topic"
-                            id="profileTopic"
```
Comment (SimonBrandner) [inline src/components/views/room_settings/RoomProfileSettings.tsx:274]: Why remove the `id`?

Comment (luixxiul) [reply src/components/views/room_settings/RoomProfileSettings.tsx:274]: Ah right, this is used for `NewRoomIntro` and should not be removed. Thanks!

File: src/components/views/room_settings/RoomProfileSettings.tsx [hunk 2]
```diff
@@ -270,8 +271,11 @@ export default class RoomProfileSettings extends React.Component<IProps, IState>
...
+                            className={classNames(
```
Comment (SimonBrandner) [inline src/components/views/room_settings/RoomProfileSettings.tsx:274]: I don't think we should use `classNames` here as it doesn't' seem to be very necessary

Comment (luixxiul) [reply src/components/views/room_settings/RoomProfileSettings.tsx:274]: It is for style lint (max length on one line).

**Review by SimonBrandner — APPROVED** [review]

**Pull request merged**